Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelNetworkTests.swift

```diff
@@ -0,0 +1,116 @@
+import NetworkService
+import XCTest
+
+final class SitterProfileViewModelNetworkTests: XCTestCase {
+    private var viewModel: SitterProfileViewModel!
+
+    override func setUp() {
+        super.setUp()
+        URLProtocol.registerClass(MockURLProtocol.self)
+        viewModel = SitterProfileViewModel()
+    }
+
+    override func tearDown() {
+        URLProtocol.unregisterClass(MockURLProtocol.self)
+        viewModel = nil
+        super.tearDown()
+    }
+
+    func testSuccessfulUploadDoesNotTriggerIsErrorOccurred() async throws {
+        // Given
+        let expectedStatusCode = 200
```

Comment:
Why do you create the constant, if we use it only once and don't check anywhere? Your previous version was good.